### ISSUE TITLE
Compute Partial Dependence for stacked ensemble pipelines

### ISSUE THREAD

**Issue by freddyaboulton** [alteryx/evalml#2121]
Repro

```python
from evalml.pipelines import StackedEnsembleClassifier, BinaryClassificationPipeline
from evalml.pipelines.utils import make_pipeline_from_components
from evalml.demos import load_breast_cancer
from evalml.model_understanding import partial_dependence
import pytest

class RfPipeline(BinaryClassificationPipeline):
    component_graph = ['Random Forest Classifier']

class XgbPipeline(BinaryClassificationPipeline):
    component_graph = ['XGBoost Classifier']

X, y = load_breast_cancer()

rf = RfPipeline({})
xgb = XgbPipeline({})

ensemble = make_pipeline_from_components([StackedEnsembleClassifier([rf, xgb])], problem_type="binary")

ensemble.fit(X, y)

with pytest.raises(NotImplementedError):
    partial_dependence(ensemble, X, features='mean radius', grid_resolution=15)
```

The issue is because `sk_wrapped_estimator` is accessing the feature importance which raises a NotImplementedError for stacked ensembles.

**Comment by dsherry** [comment]
Thanks @freddyaboulton . I think that #1930 + #1281  would make this easier to fix, right? But we will be able to patch it before those issues are complete. Does that sound correct to you?

**Comment by freddyaboulton** [comment]
Closed by #2712
